sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/DownloadCBDE.targets

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Targets to download the CBDE binaries if required. 
+  
+       The files will be downloaded automatically if they are not found locally.
+       
+       To force the files to be downloaded even if they exist, set the build property "UpdateCBDE" to true.
+  
+  -->
+
+  <PropertyGroup>
+    <RelativeCBDEFolder>CBDE\</RelativeCBDEFolder>
+    <CBDEFolder>$(MSBuildProjectDirectory)\$(RelativeCBDEFolder)</CBDEFolder>
+    <SpecificCBDEVersion>1.0.0.13341</SpecificCBDEVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <CBDEUrls Include="http://repox.jfrog.io/repox/sonarsource-public-builds/org/sonarsource/cbde/$(SpecificCBDEVersion)/cbde-windows.$(SpecificCBDEVersion).zip" >
+      <SubFolder>windows\</SubFolder>
+      <MainExe>dotnet-symbolic-execution.exe</MainExe>
+      <ZipFileName>cbde-windows.zip</ZipFileName>
+    </CBDEUrls>
+    <!-- Only Windows is supported for now
```

Comment:
I guess the best is to remove this ? (having commented out logic is not great)

Reply:
Code referenced in https://github.com/SonarSource/SonarCBDE/issues/67